roles/network_plugin/multus/templates/multus-daemonset.yml.j2

```diff
@@ -52,6 +59,7 @@ spec:
 {% if container_manager == 'crio' %}
         - name: run
           mountPath: {{ multus_cni_run_dir }}
+          mountPropagation: HostToContainer
```

Comment:
I could not get the reason why here needs to specify `mountPropagation: HostToContainer` here.
Can I get the point?